api/kessel/inventory/v1beta1/k8s_cluster.proto

```diff
@@ -11,6 +11,7 @@ import "kessel/inventory/v1beta1/reporter_data.proto";
 option go_package = "github.com/project-kessel/inventory-api/api/kessel/inventory/v1beta1";
 option java_multiple_files = true;
 option java_package = "org.project_kessel.api.inventory.v1beta1";
+option java_outer_classname = "K8sClusterProto";
```

Comment:
Why the "Proto" suffix? Can we just name these `"K8sCluster"` (etc)?

Reply:
If I set the option java_outer_classname = "K8sCluster" 
getting error:
```/Users/akoserwa/ci-work/inventory-client-java/src/main/proto/kessel/inventory/v1/health.proto [0:0]: --java_out: kessel/inventory/v1beta1/k8s_cluster.proto: kessel/inventory/v1beta1/k8s_cluster.proto: Cannot generate Java output because the file's outer class name, "K8sCluster", matches the name of one of the types declared inside it.  Please either rename the type or use the java_outer_classname option to specify a different outer class name for the .proto file.

```
I found some examples where the "Proto/Protos" suffix is used as general practice. I am not sure whether it is the best practice or convention.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2xla/host_compute_metadata.proto#L9

Reply:
I see. The java generator creates a class for the entire file using the name of the .proto file. It then encapsulates the messages in the proto file as inner classes of that "outer class."

Since there's precedent, I say we just go with the suffix and move on.